Makefile

```diff
@@ -92,7 +92,7 @@ endif
 test-dev: verify test-browser-dev test-webpack
 
 deploy:
-	webpack --bail --config build/deploy/webpack.config.js
+	webpack --bail --config build/deploy/webpack.deploy.config.js --define process.env.NODE_ENV="'production'"
```

Comment:
is there a reason we can't just `export NODE_ENV=production` on the previous line